googleMapsApiModule.py

```diff
@@ -17,6 +17,7 @@ def direction(origin, destination):
     result = gmaps.directions(origin, destination)
     address = f'origin={origin}&destination={destination}'
     result_url = f'{BASE_URL["direction"]}&{address.lower().replace(" ", "+")}'
+    return result_url
     logging.debug(result_url)
     webbrowser.open_new(result_url)
```

Comment:
Hmm, but now link won't be opened if someone wants to use locally right? Rather, can you have these return statements at the end of the function and try mocking the webbrowser.open() function within the tests 😄

Reply:
Yes, this is very important. Make sure the same is taken care of in all methods.

Reply:
Resolved. Please review.